stake-pool/single-pool/src/processor.rs

```diff
@@ -891,11 +915,8 @@ impl Processor {
         }
 
         let vote_address_str = vote_account_address.to_string();
-        let token_name = format!(
-            "SPL S-Val Token ({}...{})",
-            &vote_address_str[0..4],
-            &vote_address_str[vote_address_str.len().saturating_sub(4)..vote_address_str.len()],
-        );
+        let token_name = format!("SPL Single Pool {}", &vote_address_str[0..15]);
+        let token_symbol = format!("s-{}", &vote_address_str[0..7]);
```

Comment:
micro-nit: how about the `st` prefix, like with `stETH` or `stSOL`?
```suggestion
        let token_symbol = format!("st{}", &vote_address_str[0..7]);
```